### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by smith**
* Change "Changelog" to "Change Log"
* "available" was spelled wrong in the release notes
* "Source" was spelled wrong in the release process

ChangeLog-Entry: Fix typos in release notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Chef Server Changelog
+# Chef Server Change Log
```
Comment (tylercloke) [inline CHANGELOG.md]: Pretty sure `changelog` is generally accepted spelling.

Comment (smith) [inline CHANGELOG.md]: Meh, ok I'll change it if you want.

**Comment by smith** [comment]
@tylercloke this has been updated and rebased.

**Comment by tylercloke** [comment]
:+1:

**Comment by stevendanna** [comment]
:+1:

**Pull request merged**